src/components/Dropdown/index.tsx

```diff
@@ -66,8 +71,10 @@ export const DropdownComponent = (props: DropdownComponentProps) => {
     }, [handleSelect]);
 
     React.useEffect(() => {
-        setSelected(placeholder || convertToString(defaultPlaceholder));
-    }, [placeholder, defaultPlaceholder]);
+        if (clear === undefined || clear) {
```

Comment:
Can be replaced with:
```suggestion
        if (clear !== false) {
```

Reply:
fixed